compose/material3/material3/src/commonMain/kotlin/androidx/compose/material3/Slider.kt

```diff
@@ -289,8 +289,6 @@ fun Slider(
 ) {
     val state =
         remember(steps, valueRange) { SliderState(value, steps, onValueChangeFinished, valueRange) }
-
-    state.onValueChangeFinished = onValueChangeFinished
```

Comment:
It was removed in https://r.android.com/3446328. Reverting this line fixes our test.

@igordmn should we introduce a diff in common here?

Reply:
> Reverting this line fixes our test.

Which test?

> @igordmn should we introduce a diff in common here?

Can you rephrase the question? Why we shouldn't, as it is AOSP change?

Reply:
I see, it doesn't fix the test, it breaks the test )

Reply:
We can change commonMain if it breaks our tests.

Let's change, add TODO and create a Major ticket to upstream/create an issue in AOSP/notify the regression in the CL

This was investigated by @m-sasha

Reply:
Also, as it is local regression, we can Ignore the test to speed up the merge, and create a critical task with target 1.8.0-beta01

Reply:
https://youtrack.jetbrains.com/issue/CMP-7512